### ISSUE TITLE
Extractor : wrong SRS for vector data

### ISSUE THREAD

**Issue by bchartier** [georchestra/georchestra#1906]
I tried to extract data from a GeoServer layer coming from the dev server of GéoPicardie.
This layer is stored as a shapefile in EPSG:2154 projection.
I chose to extract it as a shapefile in WGS84 SRS.
The .proj files of the data layer and the bbox are coherent with WGS84.
The data layer coordinates seem to be totally incoherent with this projection. See the ogrinfo command result obtained with the extracted data:

```
Layer name: olv_equ_bdolv_geo_smvo_dechetterie_POINT
{{ Geometry: 3D Multi Point}}
{{ Feature Count: 24}}
{{ Extent: (626997.320938, 6889290.994452) - (704022.321241, 6951376.923891)}}
{{ Layer SRS WKT:}}
{{ GEOGCS["WGS 84",}}
{{ DATUM["World Geodetic System 1984",}}
{{ SPHEROID["WGS 84",6378137.0,298.257223563,}}
{{ AUTHORITY["EPSG","7030"]],}}
{{ AUTHORITY["EPSG","6326"]],}}
{{ PRIMEM["Greenwich",0.0,}}
{{ AUTHORITY["EPSG","8901"]],}}
{{ UNIT["degree",0.017453292519943295],}}
{{ AXIS["Geodetic longitude",EAST],}}
{{ AXIS["Geodetic latitude",NORTH],}}
{{ AUTHORITY["EPSG","4326"]]}}
```

[for Camptocamp developers, the original description of the issue is there (in french): https://jira.camptocamp.com/browse/GSPIC17-129]

**Comment by fvanderbiest** [comment]
A fix is planned for 19.04.

**Comment by groldan** [comment]
reopening to remove the WIP label
